src/js/profile/category.js

```diff
@@ -0,0 +1,43 @@
+import {Subcategory} from "./subcategory";
+
+let categoryTemplate = null;
+let profileHeader = null;
+let indicatorTemplate = null;
+let tooltipClone = null;
+
+const categoryHeaderTitleClass = '.data-category__header h2';
+const categoryHeaderDescriptionClass = '.data-category__header_description p';
+const subcategoryClass = '.data-category__indicator';
+
+export class Category {
+    constructor(category, detail, _categoryTemplate, _profileHeader, _indicatorTemplate, _tooltipClone) {
```

Comment:
I wonder whether we can stuff _indicatorTemplate and _tooltipClone into a dictionary/object instead of explicitly passing parameters. Since they are passing through category directly to subcategory perhaps it's not worth the extra effort of copying them in the constructor only to then pass them through, so something like:

 constructor(category, detail, _categoryTemplate, _profileHeader, additionalElements)

an then you create category like this:
new Category(category, detail, categoryTemplate, profileHeader, {indicatorTemplate: indicatorTemplate, tooltipClone: tooltipClone})

Subcategory would then dig around in the additional elements for what it's looking for, e.g.

class Subcategory {
    constructor(..., additionalElements) {
        this.indicatorTemplate = additonalElements.indicatorTemplate
       ....
    }
}